src/test/rgw/CMakeLists.txt

```diff
@@ -52,6 +52,15 @@ set(test_rgw_a_src test_rgw_common.cc)
 add_library(test_rgw_a STATIC ${test_rgw_a_src})
 target_link_libraries(test_rgw_a ${rgw_libs})
 
+add_executable(test_rgw_ratelimit test_ratelimit.cc)
+target_link_libraries(test_rgw_ratelimit ${rgw_libs} Boost::coroutine)
+add_executable(test_rgw_ratelimit_gc test_ratelimit_gc.cc)
+target_link_libraries(test_rgw_ratelimit_gc ${rgw_libs} Boost::coroutine)
+
+add_executable(unittest_rgw_ratelimit test_ratelimit_unittest.cc)
```

Comment:
nit: i would call this one `test_ratelimit`, and the others `bench_ratelimit{_gc}`